shift_detector/precalculations/store.py

```diff
@@ -14,12 +15,14 @@ class Store:
     def __init__(self,
                  df1: DataFrame,
                  df2: DataFrame,
+                 log_print=False,
```

Comment:
the default value differs from `Detector.__init__` Please make it consistent.

Reply:
Why does it have to be consistent? If the Store is called alone I don't want print statements. If the Store is called through the detector we show the logs by default. The user needs to specify if he don't want to see them. By doing this we only have to change the Detector calls in the tests and the user don't necessary need to know about the log_print variable because we should the logs by default.

Reply:
I'm with Jan on this one, the detetcor is the class that the user is most likely to use. Therefore it should print what it is executing. In contrast the store is not solely designed to be called on its own, hence it makes sense to not print here what he is executing.